mesonbuild/modules/windows.py

```diff
@@ -87,7 +87,7 @@ def _find_resource_compiler(self, state: 'ModuleState') -> T.Tuple[ExternalProgr
 
         for (arg, match, rc_type) in [
                 ('/?', '^.*Microsoft.*Resource Compiler.*$', ResourceCompilerType.rc),
-                ('/?', 'LLVM Resource Converter.*$', ResourceCompilerType.rc),
+                ('/?', '^.*LLVM Resource Converter.*$', ResourceCompilerType.rc),
```

Comment:
If the string we are trying to match is:
```
OVERVIEW: Resource Converter
```

How does this change help?

Reply:
yep

Reply:
I am not sure what you mean by that reply.